example-dotnet/Program.cs

```diff
@@ -11,8 +22,134 @@ static void Main(string[] args)
             Say.hello("example");
             Console.WriteLine("Hello World!");
 
+	    // Usage of BCrypt.Net API
+
             var hashedPassword = BCrypt.Net.BCrypt.HashPassword("my-secret-password");
             BCrypt.Net.BCrypt.Verify("wrong-password", hashedPassword);
+
+            // Usage of SharpZipLib API
+
+	    // This can trigger an exploit, see:
+	    // https://github.com/icsharpcode/SharpZipLib/issues/232
+	    // The usage of SharpZipLib API here will be reported as
+	    // may be reaching one of the methods affected by an
+	    // update from vulnerable version to the safe version.

```

Comment:
This paragraph is subsumed by the one below.

Reply:
I've replaced all the lengthy paragraphs with tables.